upstairs/src/lib.rs

```diff
@@ -2903,6 +2926,14 @@ impl Upstairs {
          */
         #[cfg(not(test))]
         assert_eq!(opt.target.len(), 3);
+        /*
+         * The repair port is the downstairs target port + 4000
```

Comment:
Weren't we going to use even + odd ports for this? We didn't record the meeting so I can't say for sure but 1 instead of 4000 would allow the agent to allocate these pairs together.

Reply:
Not that the agent can't allocate X and X+4000 together, just that it makes packing them easier

Reply:
We decided to use the Fibonacci sequence to select the next port number,   
We did record the meeting, you can find the link to it in the calendar entry for Monday, after the meeting recording is finished, google goes back in time and adds the recording link to the meeting info.

We talk about both +1 and +4000,  +1 at the start of the meeting, and +4000 around 5:00 into the meeting.

The reason I chose +4000 right now is to not have to go move all the tests we have written to skip ports, right now
they all just create downstairs with sequential port numbers.   I do plan to do that, but was hoping to kick that down the road a little longer.  Making the REPAIR_PORT_OFFSET a const allows me to come back later and switch it to +1 after updating all the tests.

Reply:
:+1: This sounds good. For reference, when I said "We didn't record the meeting" I was referring to the in-person one on Friday of the meetup week.

Reply:
Got it about meetings.